library/foundation/src/main/java/com/epmedu/animeal/foundation/animation/VerticalAnimatedVisibility.kt

```diff
@@ -0,0 +1,20 @@
+package com.epmedu.animeal.foundation.animation
+
+import androidx.compose.animation.AnimatedVisibility
+import androidx.compose.animation.AnimatedVisibilityScope
+import androidx.compose.animation.slideInVertically
+import androidx.compose.animation.slideOutVertically
+import androidx.compose.runtime.Composable
+
+@Composable
+fun VerticalAnimatedVisibility(
```

Comment:
`SlideInSlideOutAnimatedVisibility`?

Reply:
Renamed to `VerticalSlideAnimatedVisibility`